Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -638,9 +664,10 @@ def latest_version(
         next if strategy.blank?
 
         strategy_data = strategy.find_versions(
```

Comment:
One alternative to the approach taken above is to build a hash of args for this method ahead of time and pass it as e.g. `**livecheck_strategy_args`, so that only the parameters expected by the strategy will be included.

Reply:
I think this idea is outside of the scope of this PR but, since we're on the topic, one thing to consider is where we would build the hash of args. It's specific to each strategy, so it seems like we would either introduce more strategy-specific logic into `#latest_version` (which I'm wary of doing) or we would add a method to all the strategies that takes a `**strategy_args` argument and returns only the arguments that its `#find_versions` method expects.

Unless I'm mistaken, the only benefit is that it would allow us to remove the `unused` parameter from the `#find_versions` methods. Maybe you have something more elegant in mind but, simply going off of the potential approaches mentioned above, I'm not sure this would be worth the tradeoff.